front/app/containers/ProjectsShowPage/shared/volunteering/CauseCard.tsx

```diff
@@ -250,7 +250,12 @@ const CauseCard = memo<Props>(({ cause, className }) => {
           <Title>
             <T value={cause.attributes.title_multiloc} />
           </Title>
-
+          <Label hidden>
```

Comment:
Your instinct is right but labels are used with inputs, normally. In this case, I think it's best to use the `ScreenReaderOnly` helper instead of a hidden label to achieve the same result.